src/main/java/com/nerodesk/om/mock/MkFriends.java

```diff
@@ -83,16 +97,19 @@ public boolean leader() throws IOException {
         assert this.dir != null;
         assert this.user != null;
         assert this.label != null;
```

Comment:
@alevohin hmm.. I copied it from some other class, can't remember now. I guess I could just remove them, right? Useless.